template/app/src/main/java/com/xmartlabs/template/ui/recyclerfragmentexample/RecyclerExampleFragment.java

```diff
@@ -0,0 +1,44 @@
+package com.xmartlabs.template.ui.recyclerfragmentexample;
+
+import android.support.annotation.NonNull;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+
+import com.annimon.stream.IntStream;
+import com.hannesdorfmann.fragmentargs.annotation.FragmentWithArgs;
+import com.xmartlabs.template.R;
+import com.xmartlabs.template.ui.common.TemplateFragment;
+
+import java.util.List;
+import java.util.Locale;
+
+import butterknife.BindView;
+
+@FragmentWithArgs
+public class RecyclerExampleFragment extends TemplateFragment<RecyclerExampleView, RecyclerExamplePresenter>
```

Comment:
in my opinion we should make a `baseListfragment`, it should get the adapter type from a generic parameter.. maybe in the future